City-of-Helsinki/drupal-helfi-strategia
# UHF-4064: Update hdbt and hdbt_admin themes, add correct settings for…

# Correct koro-figure and color palette [UHF-4064](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4064)
Switch the color palette and koro-figure to correct ones and update hdbt and hdbt_admin themes so that the settings are applied correctly.

## What was done
* Updated hdbt and hdbt_admin themes
* Added correct site settings to configurations
* Added also some configuration that was missing from codebase.

## How to install
* Checkout this branch.
* Run `composer install`
* Run `make drush-cr && make drush-cim && make drush-cr`

## How to test
* Go to `/admin/tools/site-settings` and make sure that correct koro-figure and color palette is selected. NOTICE: these can be in config ignore as well so you might not see anything change here and if that is the case change the values manually.
* Correct values are Gold & Basic.
* Check that the site uses the selected values correctly (footer koro-figure should be Basic for example and newly created content should use the Gold palettes colors).
